### ISSUE TITLE
WAMP 3.1.9

### ISSUE THREAD

**Issue by d95gas** [Frecuencio/sqlbuddy-php7#7]
Installed latest version of SQL Buddy into WAMP 3.1.9 which is running
PHP 7.2.18
MYSQL 5.7.26
Apache 2.4.39

Still getting issue with response:

You don't have permission to access /sqlbuddy/ on this server.
Apache/2.4.39 (Win64) PHP/7.2.18 Server at 127.0.0.1 Port 80

Tried various changes in .conf files, but nothing seem to affect it.

Its a shame, as I really love using this to look after my databases

**Comment by IP-CAM** [comment]
SQL Buddy does unfortunately only function up to PHP 7.0, as I found out.

**Comment by d95gas** [comment]
I have managed to get it working fine with the latest version of WAMP and this updated SQLBUDDY file.
For future ref here is how I completed it.

My Install is located at **C:\WAMP**

I extracted this GIT file and renamed the folder "sqlbuddy" and dropped the folder into the "Apps" directory of WAMP.

I then created a "**sqlbuddy.conf**" file and inserted the following into it:

Alias /sqlbuddy/ "c:/wamp/apps/sqlbuddy-php7/" 

<Directory "c:/wamp/apps/sqlbuddy-php7/">
    Options Indexes FollowSymLinks MultiViews
    AllowOverride all
        Order allow,deny
    Allow from all
</Directory>

Then running WAMP, selected the option:
"Your Virtual Hosts" -> "**Virtualhost Management**"

From the window that pops up, populate it with the details of the location of the SQLBUDDY directory, etc.

Restarted WAMP Services.   Then went into VirtualHosts and there is SQLBUDDY, selected it, and hey presto it runs perfectly, and I am now using it to work with my databases.

Very happy with the outcome as "sqlbuddy" is my favourite interface to work with.
